srcpkgs/hyprpicker/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'hyprpicker'
+pkgname=hyprpicker
+version=0.1.1
+revision=1
+hostmakedepends="pkg-config cmake ninja wayland-devel git"
+makedepends="wayland-devel wayland-protocols cairo-devel pango-devel libjpeg-turbo-devel wlroots-devel"
+short_desc="Wlroots-compatible Wayland color picker that does not suck"
```

Comment:
wlroots is the project name, this shouldn't be capitalized, even if the linter says so.

Reply:
Makes sense, thanks.